scripts/Connector/Connector_Molex/conn_molex_micro-fit-3.0_tht_side_dual_row.py

```diff
@@ -232,4 +273,5 @@ def generate_one_footprint(pins, configuration):
     configuration['kicad4_compatible'] = args.kicad4_compatible
 
     for pincount in pins_per_row_range:
-        generate_one_footprint(pincount, configuration)
+        for type in types:
```

Comment:
`type` is a python build-in function. a rather bad choice for a variable name.
perhaps use variant?

Reply:
Good point. Will fix.